docs/orchestrating-elastic-stack-applications/upgrading-stack.asciidoc

```diff
@@ -9,6 +9,8 @@ endif::[]
 
 The operator can safely perform upgrades to newer versions of the various Elastic Stack resources.
 
-Follow the instructions in the link:https://www.elastic.co/guide/en/elastic-stack/current/upgrading-elastic-stack.html[Elasticsearch documentation]. Make sure that your cluster is compatible with the target version, take backups, and follow the specific upgrade instructions for each resource type, especially the order in which the upgrade should be carried out. When you are ready, modify the `version` field in the resource spec to the desired stack version and the operator will start the upgrade process automatically.
+Follow the instructions in the link:https://www.elastic.co/guide/en/elastic-stack/current/upgrading-elastic-stack.html[Elasticsearch documentation]. Make sure that your cluster is compatible with the target version, take backups, and follow the specific upgrade instructions for each resource type. When you are ready, modify the `version` field in the resource spec to the desired stack version and the operator will start the upgrade process automatically.
+
+ECK will make sure that Elastic Stack resources are upgraded in the correct order. Upgrades to dependent stack resources will be delayed until the dependency is upgraded. For example, the Kibana upgrade will only then be rolled out when the associated Elasticsearch cluster has been upgraded.
```

Comment:
I'm not sure about my suggestion, but something sounds off to me in the original. Tagging @alaudazzi :)
```suggestion
ECK will make sure that Elastic Stack resources are upgraded in the correct order. Upgrades to dependent stack resources will be delayed until the dependency is upgraded. For example, the Kibana upgrade will be rolled out only when the associated Elasticsearch cluster has been upgraded.
```

Reply:
Your suggestion sounds good @david-kow. I would also remove the future tense.